ui/core/individual_sim_ui.tsx

```diff
@@ -525,6 +531,36 @@ export abstract class IndividualSimUI<SpecType extends Spec> extends SimUI {
 		});
 	}
 
+	static updateProtoVersion(settingsProto: IndividualSimSettings) {
+		if (!(settingsProto.apiVersion < CURRENT_API_VERSION)) {
+			return;
+		}
+		// Define the conversion map between Stat/ PseudoStat schemas.
+		const conversionMap: ProtoConversionMap<IndividualSimSettings> = new Map([
+			[
+				1,
+				(oldProto: IndividualSimSettings) => {
+					oldProto.apiVersion = 1;
```

Comment:
These should be changed to 2 once you increment the `current_version_number` option.